SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCloudLauncherSettings.h

```diff
@@ -83,6 +106,32 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorCloudLauncherSettings : public UObje
 			: SnapshotPath.FilePath;
 	}
 
+	void SetPrimaryRegionCode(const ERegionCode::Type RegionCode);
+	FORCEINLINE FText GetPrimaryRegionCode() const
+	{
+		UEnum* pEnum = FindObject<UEnum>(ANY_PACKAGE, TEXT("ERegionCode"), true);
+
+		if (!pEnum || !IsRegionCodeValid(PrimaryDeploymentRegionCode))
```

Comment:
Do we need to check for the enum validity here? isn't that being done in the function already?

Reply:
This is piece of code is also used for the GUI - so if there's a corrupt/invalid settings file it will show "Invalid" for the region code.